kie-gwthelper-maven-plugin/src/main/java/org/kie/maven/gwthelper/plugin/AddSourceMojo.java

```diff
@@ -0,0 +1,191 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kie.maven.gwthelper.plugin;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+import org.apache.maven.plugin.AbstractMojo;
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugins.annotations.LifecyclePhase;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+import org.apache.maven.project.MavenProject;
+import org.codehaus.plexus.util.StringUtils;
+
+/**
+ * Add more source directories to the POM.
+ */
+@Mojo(name = "add-source", defaultPhase = LifecyclePhase.GENERATE_SOURCES, threadSafe = true)
+public class AddSourceMojo  extends AbstractMojo {
+
+    private final static String SRC_MAIN_JAVA = "/src/main/java".replace("/", System.getProperty("file.separator"));
+    private final static String SRC_MAIN_RESOURCES = "/src/main/resources".replace("/", System.getProperty("file.separator"));
+
+    /**
+     * Comma-separated additional source directories.
+     * @since 0.1
```

Comment:
Should be updated to the actual plugin version or removed.

Reply:
_As far as I know, the since tag should be used to define in which version you added the method, class, etc, while version tag should be the current version of the release or file in question. Anyway, removing._

Reply:
Yes that makes sense. Feel free to keep it there, I just think the value should reflect actual version of the plugin, which would be `7.19.0.Final` if it's merged to master now.

On the other hand, the plugin is not intended for public use (if I understand correctly) so we don't have to be that punctual about documenting its parameters. In that case, leaving this information out makes sense too.

Reply:
To be honest, I think that plugin could be useful for others. Basically, this address the GWT multimodule issue without IDE plugins. The main limitation of the maven gwt plugins (the old as the tbroyer one as far as I know) is the fact that it is hard to work with code that is "elsewhere" in the filesystem; the "addsources" (standard maven helper plugin) was a first solution useful for limited cases; I hope this could have a broader usage.